lib/build.api.js

```diff
@@ -32,8 +32,7 @@ class Build {
       symlink: true,
       delPatterns: ['*', '.*'],
       excludeLinks: [],
-      addLinks: [],
-      makeFile: this.getAbsolutePath('drupal.make.yml')
+      addLinks: []
```

Comment:
I think makeFile should be come `make` and should have the default value of `false`, since it is a property that we need to use, but may not immediately have a value for it.